cmd/kubebuilder/generate/generate.go

```diff
@@ -100,12 +122,6 @@ func RunGenerate(cmd *cobra.Command, args []string) {
 
 	util.GetCopyright(copyright)
```

Comment:
This also needs to be moved at the start of the fn because it is being used by the `controller-gen`